graylog2-web-interface/src/contexts/HotkeysContext.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import type { Hotkey } from 'react-hotkeys-hook/dist/types';
+import type { Options } from 'react-hotkeys-hook';
+import type Immutable from 'immutable';
+
+import { singleton } from 'logic/singleton';
+
+export type ScopeName = 'view' | 'search' | 'dashboard';
```

Comment:
It would be nice to infer the scope names based on the `hotKeysCollection`.

Reply:
Having another look it is fine the way it is, because we use this as a type for the hotkeys collection.